<!doctype html>
<html>
<head>
  <meta charset=utf-8 />
  <title> Ben Moore's CV</title>
  <link href="media/cv-screen.css" type="text/css" rel="stylesheet" media="screen">
  <link href="media/cv-print.css" type="text/css" rel="stylesheet" media="print">
  <link href="media/academicons.min.css" rel="stylesheet">

  <!-- imports -->
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
  <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,700' rel='stylesheet' type='text/css'>
</head>
<body>
  <div id="main">
    <div id="content">
    <h1 id="benjamin-l-moore">Benjamin L. Moore</h1>
<p>Bioinformatics PhD and data scientist</p>

<div id="webaddress">
<a href="mailto:ben@blm.io">ben@blm.io</a>
|
<i class="fa fa-github"></i> <a href="http://github.com/blmoore">blmoore</a>
|
<i class="fa fa-twitter"></i> <a href="http://twitter.com/benjaminlmoore">benjaminlmoore</a>
</div>

<h2 id="currently">Currently</h2>

<p>I’m a Bioinformatics Scientist at <a href="http://www.illumina.com/">Illumina</a>, based at their R&amp;D site near Cambridge, UK. Here I develop software and use data science, machine learning and statistical programming to analyse large volumes of high-throughput sequencing data.</p>

<h2 id="employment">Employment</h2>

<p><code class="language-plaintext highlighter-rouge">2020-</code> 
<strong>Illumina, Inc.</strong> Senior Bioinformatics scientist (cloud software engineer and solutions architect)</p>

<p><code class="language-plaintext highlighter-rouge">2015-20</code> 
<strong>Illumina, Inc.</strong> Bioinformatics scientist (data scientist and statistical programmer)</p>

<h2 id="education">Education</h2>

<p><code class="language-plaintext highlighter-rouge">2012-15</code>
<strong>University of Edinburgh</strong> Bioinformatics PhD (supervisors: Colin Semple and Stuart Aitken). My thesis was titled <em>Unravelling higher order chromatin organisation through statistical analysis</em> and is available through the <a href="https://www.era.lib.ed.ac.uk/handle/1842/22906">Edinburgh Research Archive</a>.</p>

<p><code class="language-plaintext highlighter-rouge">2011-12</code>
<strong>Imperial College London</strong> MSc Bioinformatics and Theoretical Systems Biology (<em>Distinction</em>)</p>

<p><code class="language-plaintext highlighter-rouge">2008-11</code>
<strong>University of York</strong> BSc Biology (<em>First class honours</em>)</p>

<h2 id="presentations">Presentations</h2>

<h3 id="tutorials">Tutorials</h3>

<p><code class="language-plaintext highlighter-rouge">2014</code>
WikiProject Computational Biology half-day tutorial (<em>ISMB</em>, Boston, USA)</p>

<h3 id="talks">Talks</h3>

<p><code class="language-plaintext highlighter-rouge">2015</code>
EdinbR: The Edinburgh R usergroup (<em>various</em>)</p>

<p>MRC IGMM research seminars and section meetings (<em>various</em>)</p>

<p><code class="language-plaintext highlighter-rouge">2014</code>
Unravelling higher order chromatin structure (<em>4<sup>th</sup> Edinburgh Bioinformatics meeting</em>, Edinburgh, UK)</p>

<p>blogR: pop data analysis and R for the web (<em>Psychology R-users</em>, Edinburgh, UK)</p>

<h3 id="posters">Posters</h3>

<p><code class="language-plaintext highlighter-rouge">2015</code>
<em>Keystone Epigenomics</em> (Keystone CO, USA)</p>

<p><code class="language-plaintext highlighter-rouge">2014</code>
<em>ISMB</em> (Boston MA, USA)</p>

<p><em>Genome informatics</em> (Cambridge, UK)</p>

<p><em>Chromatin: From nucleosomes to chromosomes</em> (Cambridge, UK)</p>

<p><code class="language-plaintext highlighter-rouge">2013</code>
<em>Genome informatics</em> (CSHL NY, USA)</p>

<p><em>EpiGeneSys</em> (Cambridge, UK)</p>

<h2 id="publications">Publications</h2>

<h3 id="journal-articles">Journal articles</h3>
<p><code class="language-plaintext highlighter-rouge">2019</code>
Krusche P, Trigg L, Boutros PC, Mason CE, De La Vega FM, <strong>Moore BL</strong> <em>et al.</em> Best practices for benchmarking germline small-variant calls in human genomes. <em>Nature Biotechnology</em>, 37:555–56.</p>

<p><code class="language-plaintext highlighter-rouge">2017</code>
Peluso S, Douglas A, Hill A, De Angelis C, <strong>Moore BL</strong> <em>et al.</em> Fibroblast growth factors (FGFs) prime the limb specific Shh enhancer for chromatin changes that balance histone acetylation mediated by E26 transformation-specific (ETS) factors. <em>eLife</em>, 6:e28590.</p>

<p>Eberle MA, Fritzilas E, Krusche P, Källberg M, <strong>Moore BL</strong> <em>et al.</em> A reference data set of 5.4 million phased human variants validated by genetic inheritance from sequencing a three-generation 17-member pedigree. <em>Genome Research</em>, 27:157-164.</p>

<p><code class="language-plaintext highlighter-rouge">2016</code>
Yu N, Nützmann H-W, MacDonald JT, <strong>Moore B</strong> <em>et al.</em> Delineation of metabolic gene clusters in plant genomes by chromatin signatures. <em>Nucleic Acids Research</em>, 44(5):2255-2265.</p>

<p>Welch L, Brooksbank C, Schwartz R, Morgan SL, Gaeta B, Kilpatrick AM, Meitchen D, <strong>Moore BL</strong> <em>et al.</em> Applying, Evaluating and Refining Bioinformatics Core Competencies. <em>PLoS Compututational Biology</em>, 12(5): e1004943.</p>

<p><code class="language-plaintext highlighter-rouge">2015</code>
<strong>Moore BL</strong> <em>et al.</em> Integrative modeling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. <em>Genome Biology</em>, 16:110.</p>

<p>Fraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, <strong>Moore BL</strong> <em>et al.</em> Hierarchical folding and reorganization of chromosomes are linked to transcriptional changes in cellular differentiation. <em>Molecular systems biology</em>, 11:852.</p>

<p><code class="language-plaintext highlighter-rouge">2013</code>
<strong>Moore BL</strong> <em>et al.</em> High–quality protein backbone reconstruction from alpha carbons using Gaussian mixture models. <em>Journal of computational chemisty</em>, 34(22):1881-9.</p>

<p><code class="language-plaintext highlighter-rouge">2011</code>
Gardner PP, Daub J, Tate J, <strong>Moore BL</strong> <em>et al.</em> Rfam: Wikipedia, clans and the “decimal” release. <em>Nucleic acids research</em>, 39(S1):D141-5.</p>

<h3 id="book-chapters">Book chapters</h3>

<p><code class="language-plaintext highlighter-rouge">2017</code>
<strong>Moore BL</strong>, Semple CA. Crossing Borders: Modeling Approaches to Understand Chromatin Domains and Their Boundaries. <em>Epigenetics and Systems Biology</em>, 3:45-67.</p>

<h2 id="non-academic">Non-academic</h2>

<p><code class="language-plaintext highlighter-rouge">2014</code> Pieces in <em>The Huffington Post</em> (US): Here Are the Most Overrated and Underrated Movies of All Time; Celebrity Twitter Followers, by Gender (<a href="http://www.huffingtonpost.com/benjamin-moore/">huffingtonpost.com/benjamin-moore</a>)</p>

<p><code class="language-plaintext highlighter-rouge">2006–</code> Articles for <em>English Wikipedia</em>: European Nucleotide Archive, RNA thermometer, Toxin-antitoxin system and more (<a href="https://en.wikipedia.org/wiki/User:Ben_Moore">en.wikipedia.org/wiki/User:Ben_Moore</a>).</p>

<h2 id="technical-skills">Technical skills</h2>

<ul>
  <li>R</li>
  <li>Python</li>
  <li>Javascript (incl. Typescript, React)</li>
  <li>Go</li>
  <li>HTML and CSS</li>
  <li>LaTeX</li>
  <li>Bash</li>
  <li>Git / Mercurial</li>
  <li>UNIX</li>
  <li>Cluster computing</li>
  <li>Docker</li>
  <li>AWS</li>
</ul>

<h3 id="certifications">Certifications</h3>

<p><code class="language-plaintext highlighter-rouge">2021-24</code> <strong>AWS Certified Solutions Architect</strong> (<em>Associate</em>)</p>

<h2 id="areas-of-expertise">Areas of expertise</h2>

<ul>
  <li>Bioinformatics</li>
  <li>Statistics</li>
  <li>Machine learning</li>
  <li>Data visualisation</li>
  <li>Software development</li>
  <li>NGS and variant calling</li>
</ul>

<h2 id="projects">Projects</h2>

<h3 id="github">github</h3>

<p>My <a href="https://github.com/blmoore">personal github account</a> hosts my hobby development projects as well as listing contributions to open source tools. Examples include: shiny apps <a href="https://github.com/blmoore/blackspot">blackspot</a> and <a href="https://github.com/blmoore/shinybrot">shinybrot</a>, as well as an R package hosted by CRAN (<a href="https://github.com/blmoore/mandelbrot">mandelbrot</a>).</p>

<h3 id="blog">blog</h3>

<p>I write a data science blog at <a href="http://blm.io/blog">blm.io</a> where I apply R and Python to open datasets and write-up the results. My posts have been featured by sites including FiveThirtyEight, BuzzFeed, AVclub, io9 and more, and also led to being an invited blogger at the Huffington Post. My original wordpress blog (<a href="http://benjaminlmoore.wordpress.com">benjaminlmoore.wordpress.com</a>) has received over 150,000 unique visitors from 180 countries.</p>

<h3 id="edinbr">EdinbR</h3>

<p>I co-founded a usergroup for the R programming language in Edinburgh called EdinbR. I helped to organise our monthly meetings and built our website: <a href="http://edinbr.org">edinbr.org</a>. Our meetings attract 30-50 statisticians, data scientists and developers for talks and discussion about the R language and its applications. We’re sponsored by Revolution Analytics and have an organisational github account at <a href="https://github.com/EdinbR">github.com/EdinbR</a>.</p>

<h3 id="datarea">datarea</h3>

<p>I entered Imperial College’s <a href="https://www.imperial.ac.uk/data-science/education/summer-data-challenge/">Summer Data Challenge</a> competition, where entrants analysed a given dataset and then proposed a startup idea based on their results. I developed a modelling technique to select housing areas for investment and was awarded third place: £2,000 and startup support from Imperial Create Lab. My entry can be seen at <a href="http://blm.io/datarea">blm.io/datarea</a> and the code is shared on my github account.</p>

<h2 id="awards">Awards</h2>

<p>MRC Capacity building scholarship (PhD fees + enhanced stipend)</p>

<p>MRC Full postgraduate scholarship (MSc fees + stipend)</p>

<p>ISCB computational biology Wikipedia competition 2013 (<a href="http://dx.doi.org/10.1371/journal.pcbi.1003242">doi:10.1371/journal.pcbi.1003242</a>; 2nd place)</p>

<p>Imperial Create Lab’s “Summer data challenge” (<a href="http://summerdatachallenge.com">summerdatachallenge.com</a>; 3rd place)</p>

<p>Individual engagement grant from the Wikimedia Foundation to attend ISMB 2014</p>

<p>Training accreditation for Wikimedia UK, the national chapter of the Wikimedia Foundation</p>

<p>Front cover design selected for J. Comp. Chem. (34:22) (<a href="http://dx.doi.org/10.1002/jcc.23388">doi:10.1002/jcc.23388</a>)</p>

<h2 id="placements">Placements</h2>

<p><code class="language-plaintext highlighter-rouge">2010-11</code>
<strong>Wellcome Trust Sanger Institute</strong> (<em>Cambridge, UK</em>)
Worked for two summers on the Rfam database (<a href="http://rfam.xfam.org">rfam.xfam.org</a>) in the Bateman group</p>

<h2 id="links">Links</h2>

<!-- fa are fontawesome, ai are academicons -->
<ul>
  <li><i class="fa fa-envelope"></i> <a href="mailto:ben@blm.io">ben@blm.io</a><br /></li>
  <li><i class="fa fa-github"></i> <a href="http://github.com/blmoore">blmoore</a><br /></li>
  <li><i class="fa fa-twitter"></i> <a href="http://twitter.com/benjaminlmoore">benjaminlmoore</a><br /></li>
  <li><i class="fa fa-linkedin"></i> <a href="https://www.linkedin.com/in/blmoore/">LinkedIn</a></li>
  <li><i class="fa fa-stack-overflow"></i> <a href="http://stackoverflow.com/users/1274516/blmoore">StackOverflow</a></li>
  <li><i class="fa fa-wikipedia"></i> <a href="https://en.wikipedia.org/wiki/User:Ben_Moore">Wikipedia Userpage</a><br /></li>
  <li><i class="ai ai-google-scholar"></i> <a href="http://scholar.google.com/citations?user=YMxsGpsAAAAJ">Google scholar</a></li>
  <li><i class="ai ai-orcid"></i> <a href="https://orcid.org/0000-0002-4074-1933">ORCiD</a></li>
  <li><i class="ai ai-figshare"></i> <a href="https://figshare.com/authors/Benjamin_Moore/99461">figshare</a></li>
</ul>

<h2 id="references">References</h2>

<p>Available on request.</p>

<!-- ### Footer

Last updated: May 2013 -->

    </div>
  </div>

  <div class="footer"></div>

</body>
</html>
